quickstart_template/101-spinnaker-aks.sh

```diff
@@ -0,0 +1,161 @@
+      subscription_id="$1";;
+    --resource_group|-rg)
+      resource_group="$1";;
+    --vault_name|-vn)
+      vault_name="$1";;
+    --storage_account_name|-san)
+      storage_account_name="$1";;
+    --storage_account_key|-sak)
+      storage_account_key="$1";;
+    --aksClusterName|-aks)
+      aksClusterName="$1";;
+    --region|-r)
+      region="$1";;
+    --artifacts_location|-al)
+      artifacts_location="$1";;
+    --sas_token|-st)
+      artifacts_location_sas_token="$1";;
+    --help|-help|-h)
+      print_usage
+      exit 13;;
+    *)
+      echo "ERROR: Unknown argument '$key' to script '$0'" 1>&2
+      exit -1
+  esac
+  shift
+done
+
+throw_if_empty app_id $app_id
+throw_if_empty app_key $app_key
+throw_if_empty username $username
+throw_if_empty password $password
+throw_if_empty tenant_id $tenant_id
+throw_if_empty subscription_id $subscription_id
+throw_if_empty resource_group $resource_group
+throw_if_empty vault_name $vault_name
+throw_if_empty storage_account_name $storage_account_name
+throw_if_empty storage_account_key $storage_account_key
+throw_if_empty region $region
+throw_if_empty aksClusterName $aksClusterName
+
+#install az and hal
+install_az
+run_util_script "spinnaker/install_halyard/install_halyard.sh" -san "$storage_account_name" -sak "$storage_account_key" -u "$username"
+
+#get-credentials from aks
+az login --service-principal -u $app_id -p $app_key -t $tenant_id 
+az aks get-credentials --resource-group $resource_group --name $aksClusterName -f /home/$username/.kube/config 
+chmod 777 /home/$username/.kube/config
+
+#install kubectl
+curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
+chmod +x ./kubectl
+sudo mv ./kubectl /usr/local/bin/kubectl
+
+# Configure Azure provider for Spinnaker
+echo "$app_key" | hal config provider azure account add my-azure-account \
+  --client-id "$app_id" \
+  --tenant-id "$tenant_id" \
+  --subscription-id "$subscription_id" \
+  --default-key-vault "$vault_name" \
+  --default-resource-group "$resource_group" \
+  --packer-resource-group "$resource_group" \
+  --app-key
+
+#change region if region not in eastus or westus
+if [ "$region" != eastus ] && [ "$region" != westus ]; then
+hal config provider azure account edit my-azure-account \
+  --regions "eastus","westus","$region" 
+fi
+hal config provider azure enable
+
+# Configure kubernetes provider for Spinnaker
+echo "$app_key" | hal config provider kubernetes account add my-k8s-v2-account \
+  --provider-version v2 \
+  --context $aksClusterName
+hal config provider kubernetes account edit my-k8s-v2-account --kubeconfig-file /home/$username/.kube/config
+hal config provider kubernetes enable
+hal config features edit --artifacts true
+hal config deploy edit --type distributed --account-name my-k8s-v2-account
+
+# Deploy Spinnaker to aks
+hal config version edit --version 1.13.8
```

Comment:
I believe this line has been covered by your #120 that executes “install_haryard.sh”, because the latter already has the logic of “[hal edit version xxx](https://github.com/Azure/azure-devops-utils/blob/master/spinnaker/install_halyard/install_halyard.sh#L56)”. The latter always use the latest released version. So I’m suggesting removing your line here. I know as if we are running into issues using the latest version, whatever let’s still use that which is more formal.